### ISSUE TITLE
v0.11.1-i386 won't start on Elementary Luna (Ubuntu 12.04)

### ISSUE THREAD

**Issue by shumtravi** [rhiokim/haroopad#333]
When i trying to start haroopad from terminal i'v got a message:

> [2853:0412/223857:INFO:CONSOLE(1)] "Uncaught SyntaxError: Unexpected token ﻿", source: file:///tmp/.org.chromium.Chromium.wkSmON/index.html (1)

**Comment by rhiokim** [comment]
Sorry I can not expect about this issue. But you can get other information here.
- https://github.com/rhiokim/haroopad/issues/265

**Comment by narma** [comment]
I have same issue

```
$ haroopad 
[13191:0527/075504:INFO:CONSOLE(1)] "Uncaught SyntaxError: Unexpected token ", source: file:///tmp/.org.chromium.Chromium.6O3YKk/index.html (1)
```

app version 0.12.0
Linux 64-bit, Arch, ru_RU.UTF-8 locale

But it starts normally if I launch app with C locale: LC_ALL=C haroopad

**Comment by rhiokim** [comment]
Does 0.12.2 works well for you?

**Comment by narma** [comment]
Same issue with 0.12.2

**Comment by rhiokim** [comment]
@narma thanks response. 
It'll be fixed with v0.13.
